src/Css/Style.php

```diff
@@ -2073,6 +2073,10 @@ protected function _set_style_side_type($style, $side, $type, $val)
 
                 $this->_props_computed[$prop] = $has_line_style ? $val_computed : 0;
             }
+        } elseif (($style === "border" || $style === "outline") && $type === "style") {
+            if (in_array($val, Style::$BORDER_STYLES)) {
```

Comment:
I prefer passing `true` by default to `in_array`, as to avoid any unexpected comparison behavior (and to be consistent with `===`). Though I guess it doesn’t really matter in this case.

Reply:
Probably not but I'll make this change anyway. If nothing else it's more consistent usage.